src/main/java/com/snack/news/controller/AdminController.java

```diff
@@ -7,9 +7,11 @@
 import lombok.AllArgsConstructor;
 import org.springframework.data.domain.Page;
 import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import java.nio.file.Watchable;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'java.nio.file.Watchable'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4554642)